### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ddprrt**

### CODE REVIEW

File: docs/_pages/download.md [hunk 1]
```diff
@@ -45,6 +43,14 @@ JS goes at the end of the `<body>` tag of your page:
 </body>
 ```
 
+## Install via npm
+
+Groundhog is available on npm:
+
+```
+npm install --save @dynatrace/groundhog
```
Comment (tomheller) [inline docs/_pages/download.md]: is it really `@dynatrace/groundhog`

Comment (ddprrt) [inline docs/_pages/download.md]: But eh kloar: https://www.npmjs.com/package/@dynatrace/groundhog

File: README.md [hunk 2]
```diff
@@ -24,19 +24,56 @@ find their way into GWT.
...
+
+```html
+<body>
+... all your content
+
+<script type="text/javascript" src="//assets.dynatrace.com/groundhog/vVERSIONNUMBER/js/main.js"></script>
+</body>
+```
+
+VERSIONNUMBER is the version that you want to install. Check the [website for more information](http://groundhog.dynalabs.io/download).
```
Comment (katrin-freihofner) [inline README.md]: I think this url is invalid -> should it be http://groundhog.dynalabs.io/doc/download/

**Comment by katrin-freihofner** [comment]
This is fine! 👍

**Pull request merged**